JenkinsfileRT_dev

```diff
@@ -0,0 +1,56 @@
+pipeline {
+    agent { label 'macos' }
+       environment {
+        HOME="${WORKSPACE}"
+        MIRAGE_DATA="/ifs/jwst/wit/mirage_data/"
+        TEST_BIGDATA="https://bytesalad.stsci.edu/artifactory/simcal"
+        CRDS_SERVER_URL = "https://jwst-crds.stsci.edu"
+        CRDS_PATH = "${WORKSPACE}/crds_cache"
+        PATH ="${WORKSPACE}/miniconda3/bin:${PATH}"
+        TMPDIR="${WORKSPACE}/tmp"
+        XDG_CACHE_HOME="${WORKSPACE}/tmp/.cache"
+    }
+
+    stages {
+    	  stage('Setup') {
+            steps {
+                deleteDir()
+                checkout scm
+                sh("curl https://repo.anaconda.com/miniconda/Miniconda3-latest-MacOSX-x86_64.sh -o installer.sh")
+                sh("bash installer.sh -b -p ${WORKSPACE}/miniconda3")
+        		sh("conda init bash")
+                sh("conda create -n romanisim python -y")
+            }
+        }
+        stage('Install') {
+            steps {
+                sh("""#!/usr/bin/env bash
+                      source $WORKSPACE/miniconda3/etc/profile.d/conda.sh
+                      conda activate romanisim
+                      pip install -e .[test] pytest-xdist
+                   """
+                )
+            }
+        }
+        stage('Test') {
+            steps {
+                sh("mkdir data")
+                sh("mkdir data/galsim-data")
+                sh("curl https://github.com/GalSim-developers/GalSim/raw/releases/2.4/examples/data/real_galaxy_catalog_23.5_example.fits -o data/galsim-data/real_galaxy_catalog_23.5_example.fits")      
+                sh("curl https://github.com/GalSim-developers/GalSim/raw/releases/2.4/examples/data/real_galaxy_catalog_23.5_example_selection.fits -o data/galsim-data/real_galaxy_catalog_23.5_example_selection.fits")      
+                sh("curl https://github.com/GalSim-developers/GalSim/raw/releases/2.4/examples/data/real_galaxy_catalog_23.5_example_fits.fits -o data/galsim-data/real_galaxy_catalog_23.5_example_fits.fits")      
+                sh("""#!/usr/bin/env bash
+                      source $WORKSPACE/miniconda3/etc/profile.d/conda.sh
+                      export WEBBPSF_PATH="/grp/jwst/ote/webbpsf-data"
```

Comment:
I am slightly worried about this; I don't control this, and when it is updated, regression tests will start to fail?

Reply:
ok, I can try reverting to a different download URL and see if that works